demo/two.js

```diff
@@ -0,0 +1,10675 @@
+/*
+MIT License
```

Comment:
I assume it is OK to distribute this MIT file along with BSD stuff, just bringing it to your attention/consideration since I'm not an expert on licences...

Reply:
This is a separate file (from [two.js](https://two.js.org/)). MIT isn't viral, nor is BSD (as far as I know) so whether virality is even a factor here, it's a moot point because of the two licenses.